apps/ffdweb-site/src/app/(homepage)/utils/getFeaturedProjects.ts

```diff
@@ -0,0 +1,6 @@
+import { getProjectData } from '@/projects/utils/getProjectData'
+
+export async function getProjectsBySlugs(slugs: string[]) {
```

Comment:
This is in a file called `getFeaturedProjects` so we either call this function `getFeaturedProjects` or if we intend to re-use `getProjectsBySlugs`, we can move it to general FFDW utils.